core-service/src/main/resources/application.properties

```diff
@@ -2,12 +2,10 @@ spring.application.name=core-service
 server.port=8080
 
 eureka.client.service-url.defaultZone=http://localhost:8761/eureka/
-eureka.client.refresh.enable=false
-eureka.client.fetch-registry=false
 
 spring.datasource.driver-class-name=org.postgresql.Driver
 spring.liquibase.enabled=true
 spring.datasource.username=postgres
```

Comment:
I would also use env for DB username and URL

Reply:
ok